canvasapi/current_user.py

```diff
@@ -150,3 +153,168 @@ def get_bookmark(self, bookmark):
             'users/self/bookmarks/{}'.format(bookmark_id)
         )
         return Bookmark(self._requester, response.json())
+
+    def get_favorite_courses(self, **kwargs):
+        """
+        Retrieve the paginated list of favorite courses for the current user.
+        If the user has not chosen any favorites,
+        then a selection of currently enrolled courses will be returned.
+
+        :calls: 'GET /api/v1/users/self/favorites/courses \
```

Comment:
Should use `` ` `` instead of `'`. There are multiple places this needs to be fixed for the docs to build properly.